test/jdk/javax/swing/JMenuBar/TestMenuMnemonicLinuxAndMac.java

```diff
@@ -57,52 +57,49 @@ public class TestMenuMnemonicOnAltPress {
     public static void main(String[] args) throws Exception {
         for (UIManager.LookAndFeelInfo laf :
                 UIManager.getInstalledLookAndFeels()) {
-            if (laf.getName().contains("GTK")) {
-                UIManager.setLookAndFeel("com.sun.java.swing.plaf.gtk.GTKLookAndFeel");
-            } else if (laf.getName().contains("Aqua")) {
-                UIManager.setLookAndFeel("com.apple.laf.AquaLookAndFeel");
-            } else {
-                continue;
+            if (laf.getName().contains("GTK") || laf.getName().contains("Aqua")) {
+                System.out.println("Testing: "+laf.getName());
+                UIManager.setLookAndFeel(laf.getClassName());
             }
```

Comment:
```suggestion
            if (laf.getName().contains("GTK") || laf.getName().contains("Aqua")) {
                System.out.println("Testing: " + laf.getName());
                UIManager.setLookAndFeel(laf.getClassName());
                break;
            }
```
Since we expect one and only one L&F, break from the loop as soon as you found a match.